src/Entity/User.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+
+class User
+{
+    private int $id;
+
+    private string $password;
+
+    private Collection $roles;
+
+    private string $cnp;
```

Comment:
why is this private? I don't see any reason to keep it private

Reply:
Not a good reason... gdpr ...